pyqtgraph/widgets/PlotWidget.py

```diff
@@ -74,11 +74,17 @@ def close(self):
         self.setParent(None)
         super(PlotWidget, self).close()
 
-    def __getattr__(self, attr):  ## implicitly wrap methods from plotItem
-        if hasattr(self.plotItem, attr):
+    ## implicitly wrap methods from plotItem
+    ## the implementation prevents recursive calls to PlotWidget.__getattr__ that lead to a 
+    ## 'maximum recursion depth exceeded' when PlotWidget is used in a multiple inheritance 
+    ## schema - i.e., class SomeClass(PlotWidget, SomeOtherClass)
+    def __getattr__(self, attr): 
+        try:
             m = getattr(self.plotItem, attr)
-            if hasattr(m, '__call__'):
-                return m
+        except:
```

Comment:
Can you specify which exception to catch here instead of the bare exception?

Reply:
Well, in fact, no need to modify the implementation of PlotWidget.\_\_getattr\_\_. Moreover, the problem has nothing to do with multiple inheritance. It comes from the fact that the   PlotWidget.\_\_getattr\_\_ tries to access the PlotItem attribute before it actually exists - generating recursive calls  PlotWidget.\_\_getattr\_\_ trying to access self.PlotItem. 
The simplest solution is to instantiate the PlotItem attribute at the very beginning of the  PlotWidget.__init__ and we are done withe the problem.

def __init__(self, parent=None, background='default', plotItem=None, **kargs):
        ## start by instantiating the plotItem attribute in order to avoid recursive 
        ## calls of PlotWidget.__getattr__ - which access self.plotItem!
        self.plotItem = None
        ...